torch/csrc/autograd/VariableTypeManual.cpp

```diff
@@ -213,14 +212,19 @@ Tensor & copy_(Tensor & self, const Tensor & src, bool non_blocking) {
   }
   {
     at::AutoNonVariableTypeMode non_var_type_mode(true);
-    self_.copy_(src_, non_blocking);
+    static auto op = c10::Dispatcher::singleton()
+      .findSchemaOrThrow("aten::copy_", "")
+      .typed<Tensor & (Tensor &, const Tensor &, bool)>();
```

Comment:
moving the static decls to the top of the function seems like better style, plus it'll also make this less of a big brick of a replacement... I'm almost worried that people handwriting special cases like this might just stick with the old style because the new looks so big and arcane